sdk/go/common/apitype/stacks.go

```diff
@@ -16,6 +16,8 @@ package apitype
 
 // StackSummary describes the state of a stack, without including its specific resources, etc.
 type StackSummary struct {
+	// ID is the the stack's ProgramID.
+	ID string `json:"id"`
```

Comment:
Should this be ProgramID? What even is a ProgramID? Does the CLI need to do anything with this data?